megalinter/reporters/GitlabCommentReporter.py

```diff
@@ -131,7 +131,7 @@ def produce_report(self):
             # Check if there is already a MegaLinter comment
             for comment in existing_comments:
                 if (
-                    "See errors details in [artifact MegaLinter reports on"
+                    "See detailed report in [MegaLinter reports"
                     in comment.body
```

Comment:
Plz can u also test `See detailed report in MegaLinter reports` , as both can happen ? ( add a `or` condition )